src/index.html

```diff
@@ -1,14 +1,27 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
 <head>
   <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+  <meta
+    name="viewport"
+    content="width=device-width,
+            user-scalable=no,
+            initial-scale=1.0,
+            maximum-scale=1.0,
+            minimum-scale=1.0"
+  >
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>Snake</title>
   <link rel="stylesheet" href="./style.css">
 </head>
 <body>
-  <h1>Snake</h1>
+  <div class="wrapper">
+    <div class="item1 item_box">1</div>
+    <div class="item2 item_box">2</div>
+    <div class="item3 item_box">3</div>
+    <div class="item4 item_box">4</div>
+    <div class="item5 item_box">5</div>
+    <div class="item6 item_box">6</div>
```

Comment:
Сначала основной класс, потом второстепенный
```suggestion
    <div class="box item-1">1</div>
    <div class="box item-2">2</div>
```